marine_sensor_msgs/msg/RadarEcho.msg

```diff
@@ -0,0 +1,6 @@
+# This message is a submessage of RadarSector and is not intended
+# to be used separately.
```

Comment:
Feels like this is missing a 2nd sentence. Maybe something like `It represents the intensities returned from a single ping`?

Reply:
(Yes, I know that then overlaps with the field comments -- I just kept stumbling over it, expecting a more complete summary. Could probably pare down the following comment.)